livekit-agents/livekit/agents/metrics/utils.py

```diff
@@ -0,0 +1,69 @@
+import logging
+from dataclasses import dataclass
+from typing import Callable
+
+from livekit.agents.metrics import PipelineMetrics
+
+
+@dataclass
+class PipelineUsageSummary:
+    llm_prompt_tokens: int = 0
+    llm_completion_tokens: int = 0
+    tts_characters_count: int = 0
+    stt_audio_duration: float = 0
+
+
+def create_metrics_logger(
+    logger: logging.Logger | None = None,
+) -> tuple[Callable[[PipelineMetrics], PipelineUsageSummary], None]:
+    if logger is None:
+        logger = logging.getLogger("")
```

Comment:
let's use the default logger of livekit-agents instead
`from ..log import logger`